docs/deployments_subscriptions_amq.md

```diff
@@ -146,43 +146,33 @@ The service hostname can be retrieved using the terminal.
 oc get addressspace <addressspace name> -o jsonpath='{.status.endpointStatuses[?(@.name=="messaging")].serviceHost
 ```
 
-The following code can be used to connect.
-
-```js
-const mqtt = require('mqtt')
-const { MQTTPubSub } = require('@aerogear/graphql-mqtt-subscriptions')
-
-const client = mqtt.connect({
-  host: '<internal host name>',
-  username: '<MessagingUser name>',
-  password: '<MessagingUser password>',
-  port: 5762,
-})
-
-const pubsub = new MQTTPubSub({ client })
+Once you have the AMQ address and credentials, open the `server/src/pubsub.ts` file and change the `mqttOptions` variable to the following.
+
+```ts
+const mqttOptions = {
```

Comment:
Actually we have env variables for this in.env
.openshift/readme.md has all info that we need copy paste. This docs is outdated.

Reply:
Let me review this once again.